main.tf

```diff
@@ -198,6 +198,7 @@ module "alb" {
     prefix  = var.alb_log_location_prefix
   }
 
+  listener_ssl_policy_default = var.alb_listener_ssl_policy_default
```

Comment:
Fix formatting, please.

Reply:
which formatting? this passes `tf fmt` and the pre-commit hook

Reply:
my bad :)